roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
```

Comment:
It is only getting used at https://github.com/ryshah/swifttool/blob/master/swifttool/client.py#L342
This sets up fabric environment for remote copying. I did try with become and become_user as swiftops, it kicked of swifttool but user argument was not available